docs/csharp/fundamentals/coding-style/snippets/coding-conventions/program.cs

```diff
@@ -5,14 +5,14 @@ namespace Coding_Conventions_Examples
     class Program
     {
         //<snippet14a>
-        public static Action<string> ActionExample1 = x => Console.WriteLine($"x is: {x}");
+        public static Action<string> actionExample1 = x => Console.WriteLine($"x is: {x}");
 
-        public static Action<string, string> ActionExample2 = (x, y) =>
+        public static Action<string, string> actionExample2 = (x, y) =>
             Console.WriteLine($"x is: {x}, y is {y}");
 
-        public static Func<string, int> FuncExample1 = x => Convert.ToInt32(x);
+        public static Func<string, int> funcExample1 = x => Convert.ToInt32(x);
 
-        public static Func<int, int, int> FuncExample2 = (x, y) => x + y;
+        public static Func<int, int, int> funcExample2 = (x, y) => x + y;
```

Comment:
These are `public static` and should follow the same naming conventions as other public APIs. PascalCasing. They were correct as they were originally.